app/Managers/PostManager.php

```diff
@@ -54,13 +54,12 @@ public function create($data, $signupId, $authenticatedUserRole = null)
             $authenticatedUserRole,
         );
 
-        // Send to Blink unless 'dont_send_to_blink' is TRUE
+        // Send to Customer.io unless 'dont_send_to_blink' is TRUE
         $shouldSendToCustomerIo = !(
             array_key_exists('dont_send_to_blink', $data) &&
             $data['dont_send_to_blink']
```

Comment:
If I remember correctly... the only instances we'd send along this parameter are from console commands? Is renaming this parameter in scope for the cleanup?

Reply:
Oh, that could absolutely be true! I would _looooove_ to rename (or even remove!) this parameter. Let me do a little digging to see where we might still be using it and make a follow-up PR to make those changes if possible.

Reply:
I'd imagine we'd still want the property/feature for backfilling, when we wouldn't want to send a confirmation for activity created months (years?) ago... or not?

Reply:
Oh just saw this! I wonder... I think in general we'd be able to handle this by excluding the post or action from messaging in Customer.io. But I also always just jump at the chance to delete code... 😈 😂

Reply:
Moving this over into #1144 for further discussion. ⏩